ExtraEnemyCustomization/Customizations/EnemyAbilities/Abilities/EMP/Inject/Inject_PlayerBackpack.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using EECustom.Customizations.EnemyAbilities.Abilities.EMP.Handlers;
+using Gear;
+using HarmonyLib;
+using Player;
+
+namespace EECustom.Customizations.EnemyAbilities.Abilities.EMP.Inject
+{
+    internal class Inject_PlayerBackpack
+    {
+        public static void Setup()
```

Comment:
This is unused

Reply:
came out it was actually needed
https://github.com/GTFO-Modding/ExtraEnemyCustomization/pull/24/commits/66b6e6216f692c503fb87aa12cab37c630566a02